CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## 0.12.0-dev
+## 0.13.0-dev
+
+### Fixed
+
+- Character `;` inside the `URI` in `OSC 8` sequence breaking the URI
```

Comment:
Don't you want to add it into v0.12? I've intentionally scheduled it there given how trivial it is.

Reply:
I don't think it's actually a significant change and I want to make the 0.12.0 release this weekend. So I'd rather not make another RC and pull in even more stuff.

I feel like 0.12.0 is good where it is and I'd only add new stuff when there is serious breakages that were introduced in this specific version.